### PR TITLE
Create ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafk

Ticket: [DM-3471](https://cumulocity.atlassian.net/browse/DM-3471)
Original PR: [5939](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5939)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Separated ASM as a plugin. (#5939)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Application security monitoring available as a plugin
```

File: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3471
+version: 1020.0.0
+---
+Separated ASM as a plugin. (#5939)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md:17]: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility in deployments, application security monitoring (ASM) has been separated from the core Dynatrace platform and is now available as a plugin. With this change, ASM can be deployed and updated independently from the Dynatrace platform, allowing for more granular control over the ASM functionality. This separation does not impact existing ASM functionality or configurations, and users can continue to use ASM as before. However, the plugin will need to be installed separately moving forward.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md (hunk 1 above)
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md:3]:
```suggestion
title: Separated Advanced software management functionality as a plugin. (#5939)
```

File: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md:17]:
```suggestion
The Advanced software management logic was previously an integral part of the default Device Management application. With this change, the Advanced software management service has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. (#5939)
```

**Review by bozhidardedov — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Separated Advanced software management functionality as a plugin. (#5939)
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md:3]:
```suggestion
title: Separated Advanced Software Management functionality as a plugin
```

File: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3471
+version: 1020.0.0
+---
+The Advanced software management logic was previously an integral part of the default Device Management application. With this change, the Advanced software management service has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. (#5939)
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md:17]:
```suggestion
The Advanced Software Management functionality was previously an integral part of the default Device Management application. With this change, the Advanced Software Management service has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. 
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@aaronraab Please review and approve.

**Pull request merged**